packages/pagination/src/elements/Pagination/Pagination.tsx

```diff
@@ -22,33 +22,34 @@ export type PAGE_TYPE = 'next' | 'page' | 'gap' | 'previous';
 
 export interface IPaginationProps extends Omit<HTMLAttributes<HTMLUListElement>, 'onChange'> {
   /**
-   * The currently selected page
+   * Sets the current page
    */
   currentPage: number;
   /**
-   * The total number of pages available
+   * Defines the total number of pages
    */
   totalPages: number;
   /**
-   * The number of pages to pad the current page with when page gaps are
-   * displayed
+   * Sets the number of pages that appear between the current page and a gap indicator
    */
   pagePadding?: number;
   /**
-   * Position for the leading and trailing gap indicator, if needed based on
-   * current and total pages
+   * Positions the leading and trailing gap indicator, based on
+   * the current and total pages
    */
   pageGap?: number;
   /**
-   * @param {Any} currentPage - The currently selected page
+   * Handles page change events
+   *
+   * @param {Any} currentPage - The current page
    */
   onChange?: (currentPage: number) => void;
   /**
-   * Apply localized labels, test attributes, etc. to individual pages.
+   * Applies localized labels, test attributes, etc. to individual pages
    *
-   * @param {PAGE_TYPE} pageType - the type of the page to transform props for;
-   *  one of: `'previous'`, `'gap'`, `'page'`, `'next'`
-   * @param {Any} props - default page props to transform
+   * @param {PAGE_TYPE} pageType The type of the page accepting the props.
```

Comment:
staying true to JSDoc, the type here should be `string` (or is it `String`, I always forget).

Reply:
Took a quick look at the docs--looks like `string`

Reply:
Would you like me to update the function signature as well?

Reply:
Hmm this actually throws off a test, too

Reply:
Only the JSDoc should change to `string`, not the signature. We want the benefit of strict typing, but documenting with [valid](https://jsdoc.app/tags-param.html) JSDoc param types (in general).